src/shared/otterizecloud/graphqlclient/generate.go

```diff
@@ -2,5 +2,5 @@ package graphqlclient
 
 import _ "github.com/suessflorian/gqlfetch"
 
-//go:generate sh -c "go run github.com/suessflorian/gqlfetch/gqlfetch --endpoint https://app.staging.otterize.com/api/graphql/v1beta > schema.graphql"
```

Comment:
revert this back to staging so that building the operator will work for users who aren't running the cloud locally.